change_log/next/FE-1219.yml

```diff
@@ -0,0 +1 @@
+New Features: "Added basic implementation of themes"
```

Comment:
Can you add a bit more detail here - eg. what functions and how to use them. Would like to get us better at producing enough detail in release notes.